### ISSUE TITLE
pip==9.0.0 - AttributeError: 'ListCommand' object has no attribute 'find_packages_latest_versions'

### ISSUE THREAD

**Issue by amusikal** [bartTC/pip-check#12]
With pip==9.0.0 and pip-check==0.2 the following exception is raised:

```
Traceback (most recent call last):
  File "/home/vagrant/.virtualenvs/foo/bin/pip-check", line 19, in <module>
    package_list = ListCommand().find_packages_latest_versions(options)
AttributeError: 'ListCommand' object has no attribute 'find_packages_latest_versions'
```

pip==8.1.2 works fine, BTW. We can downgrade pip temporarily, but if anyone has any hints on how to resolve this please let me know.

Many thanks,

**Comment by bartTC** [comment]
Thanks. I have not updated yet but will look I to it. We had the same issue with <7.0

**Comment by rmax** [comment]
Same issue here with pip 9.0.1.

**Comment by nzjrs** [comment]
yip, same here 9.x

**Comment by Kentoseth** [comment]
The problem is here: [list.py](https://github.com/pypa/pip/blob/master/src/pip/_internal/commands/list.py)

`find_packages_latest_versions()` seems to have been replaced with: `iter_packages_latest_infos()`

I was trying to hack at it, but the code may need a major overhaul.

`find_packages_latest_versions(options)` only took 1 option, but `iter_packages_latest_infos(packages, options)` is asking for 2 options.

I tried this:
```
import pip

packages = pip.get_installed_distributions()
package_list = ListCommand().iter_packages_latest_infos(packages, options)
```

This passes to the `iter_packages ...` function, but the problems re-occur here:
```
for package, update, source in package_list:
...
```

This tool does not officially work anymore (unless you are using pip <9).

**Comment by bartTC** [comment]
This is fixed in the latest version 2.0. (We require 9.x up now)
